src/i18n/locales/es/index.json

```diff
@@ -0,0 +1,337 @@
+{
+  "about": {
+    "app_name": "Pocket Sync",
+    "new_firmware": "Nuevo Pocket Firmware v{version} disponible",
+    "update_available": "¡Actualización disponible! {version}"
+  },
+  "app": {
+    "app_name": "Pocket Sync",
+    "connect_button": "Conectar con Pocket",
+    "not_pocket_tip": "Esa carpeta no se parece al sistema de archivos de Pocket. El Pocket debe inicializar la tarjeta SD al menos una vez."
+  },
+  "clean_files": {
+    "close_button": "Cerrar",
+    "delete_files_button": "Eliminar archivos",
+    "files_found": "{count, plural, =0 {No hay archivos que se puedan limpiar} one {# archivo que se pueda limpiar} other {# archivos que se puedan limpiar} } encontrado",
+    "title": "Archivos limpios"
+  },
+  "core_info": {
+    "author": "Autor",
+    "controls": {
+      "back": "Volver a la lista",
+      "install": "Instalar",
+      "required_files": "Archivos requeridos",
+      "uninstall": "Desinstalar",
+      "update": "Actualizar"
+    },
+    "inputs": "Entradas",
+    "modal": {
+      "close": "Cerrar",
+      "input_core": "Núcleo",
+      "no_settings_found": "¡No se encontraron configuraciones!",
+      "reset_all": "Resetear todo"
+    },
+    "not_in_inventory": "{coreName} no está en el inventario de núcleos",
+    "platform": {
+      "edit": "Editar",
+      "wikipedia": "Wikipedia"
+    },
+    "replaced": "Este núcleo ha sido reemplazado por <1>{replacementCore}</1>",
+    "platforms": "Plataformas",
+    "please_wait": "Por favor espere, revisando archivos...",
+    "release_date": "Fecha de lanzamiento",
+    "releases": {
+      "release_history": "Historial de lanzamientos",
+      "show_previous": "Mostrar lanzamientos anteriores"
+    },
+    "required_files": "Archivos requeridos",
+    "required_files_count": "{count} / {total} archivos",
+    "settings": "Configuración",
+    "sponsor": "Patrocinador",
+    "supports": "Soportes",
+    "supports_items": {
+      "cartridges": "Cartuchos",
+      "dock": "Base",
+      "dock_dac": "Base (Analog)",
```

Comment:
Not sure about these two - “Dock” is the product name so probably shouldn’t be translated (unless Analogue themselves translate it?)

Reply:
Ah yes, that makes sense. Wasn't sure of the context when going through all of them. I'll update accordingly.